message_ix_models/data/level.yaml

```diff
@@ -17,6 +17,9 @@ final:
 import:
   name: Imports
 
+export:
```

Comment:
I would prefer these be in alpha order.

Please also remove the corresponding commented entry at the bottom of the file.